geoips/image_utils/maps.py

```diff
@@ -234,6 +234,8 @@ def check_gridline_annotator(gridline_annotator):
                     key, required_fields
                 )
             )
+        if key == "xpadding" or key == "ypadding":
```

Comment:
Use `if key in ["xpadding", "ypadding"]` here to simplify.

Reply:
Will do!